### PR TITLE
update use tiup client access database (#5945)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #5945 to master
You can switch your code base to this Pull Request by using [git-extras](https://github.com/tj/git-extras):
```bash
# In docs-cn repo:
git pr https://github.com/pingcap/docs-cn/pull/5958
```

After apply modifications, you can push your change to this PR via:
```bash
git push git@github.com:ti-srebot/docs-cn.git pr/5958:master-6462ffec75bc
```

---

### What is changed, added or deleted? (Required)
update use tiup client access database

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/5028
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -56,10 +56,10 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
         {{< copyable "shell-regular" >}}
 
         ```shell
-        tiup playground v4.0.0 --db 2 --pd 3 --kv 3 --monitor
+        tiup playground v4.0.11 --db 2 --pd 3 --kv 3 --monitor
```
Comment (yikeke) [inline quick-start-with-tidb.md:59]: 改成 5.0 的例子，master 的先不合，等正式发版后合并

Comment (TomShawn) [inline quick-start-with-tidb.md:59]:
```suggestion
        tiup playground v5.0.0 --db 2 --pd 3 --kv 3 --monitor
```

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -56,10 +56,10 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
...
         ```
 
-        上述命令会在本地下载并启动一个 `v4.0.0` 版本的集群，`--monitor` 表示同时部署监控组件。最新版本可以通过执行 `tiup list tidb` 来查看。运行结果将显示集群的访问方式：
+        上述命令会在本地下载并启动某个版本的集群（例如 v4.0.11），`--monitor` 表示同时部署监控组件。最新版本可以通过执行 `tiup list tidb` 来查看。运行结果将显示集群的访问方式：
```
Comment (TomShawn) [inline quick-start-with-tidb.md:62]:
```suggestion
        上述命令会在本地下载并启动某个版本的集群（例如 v5.0.0），`--monitor` 表示同时部署监控组件。最新版本可以通过执行 `tiup list tidb` 来查看。运行结果将显示集群的访问方式：
```

**Pull request merged**